GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -172,4 +180,53 @@ public void onStep() {
             }
         }
     }
+
+    // mode 0: center eye; mode 1: left eye; mode 2: right eye
+    public void captureScreen(int mode, String filename) {
+        switch (mode) {
+        case 0:
+            mGVRContext.captureScreenCenter(newScreenshotCallback(filename));
+            break;
+        case 1:
+            mGVRContext.captureScreenLeft(newScreenshotCallback(filename));
+            break;
+        case 2:
+            mGVRContext.captureScreenRight(newScreenshotCallback(filename));
+            break;
+        }
+    }
+
+    private GVRScreenshotCallback newScreenshotCallback(final String filename) {
+        return new GVRScreenshotCallback() {
+
+            @Override
+            public void onScreenCaptured(Bitmap bitmap) {
+                if (bitmap != null) {
+                    File file = new File(
+                            Environment.getExternalStorageDirectory(), filename);
+                    FileOutputStream outputStream = null;
+                    try {
+                        outputStream = new FileOutputStream(file);
+                    } catch (FileNotFoundException e) {
+                        e.printStackTrace();
+                        return;
+                    }
+                    bitmap.compress(Bitmap.CompressFormat.PNG, 100,
+                            outputStream);
+                    try {
+                        outputStream.flush();
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    try {
+                        outputStream.close();
+                    } catch (IOException e) {
+                        e.printStackTrace();
```

Comment:
It should be as the following

            @Override
            public void onScreenCaptured(Bitmap bitmap) {
                if (bitmap != null) {
                    File file = new File(
                            Environment.getExternalStorageDirectory(), filename);
                    FileOutputStream outputStream = null;
                    try {
                        outputStream = new FileOutputStream(file);
                        bitmap.compress(Bitmap.CompressFormat.PNG, 100,
                                outputStream);
                    } catch (FileNotFoundException e) {
                        e.printStackTrace();
                        return;
                    } finally {
                        try {
                            outputStream.close();
                        } catch (IOException e) {
                            e.printStackTrace();
                        }
                    }
                } else {
                    Log.e("SampleActivity", "Returned Bitmap is null");
                }
            }